docs/changelog.rst

```diff
@@ -2,6 +2,16 @@
 Changelog
 *********
 
+2.1.4 (2024-12-03)
+------------------
+
+Bug fixes:
+
+- Fix a regression where trying to use the ``poetry`` plugin without poetry
```

Comment:
@medubelko is this phrasing alright?

Reply:
The phrasings fine! I would prefer if we use the format `poetry plugin` without the inline code.

Reply:
Do you mean

```
- Fix a regression where trying to use the poetry plugin without poetry
  installed on the system would give an error.
```
?

Reply:
Correct!